eval/src/vespa/eval/eval/test/reference_evaluation.cpp

```diff
@@ -135,7 +130,7 @@ struct EvalNode : public NodeVisitor {
                 }
             }
         }
-        result = ReferenceOperations::peek(param, spec, children).normalize();
+        result = ReferenceOperations::peek(spec, children).normalize();
```

Comment:
should no longer normalize here, since it is done in eval_node instead

Reply:
fixed by rebasing and resolving conflicts